src/jit/emitxarch.cpp

```diff
@@ -2901,174 +2901,283 @@ void emitter::emitInsStoreLcl(instruction ins, emitAttr attr, GenTreeLclVarCommo
 
 regNumber emitter::emitInsBinary(instruction ins, emitAttr attr, GenTree* dst, GenTree* src)
 {
-    // dst can only be a reg or modrm
-    assert(!dst->isContained() || dst->isUsedFromMemory() || instrIs3opImul(ins)); // dst on these isn't really the dst
+    // We can only have one memory operand and only src can be a constant operand
+    GenTree* memOp = nullptr;
+    GenTree* cnsOp = nullptr;
+    GenTree* otrOp = nullptr;
```

Comment:
I guess otr is shorthand for other?

Reply:
I vote for a rename (IMO, "otherOp" is fine; no need for all vars to be 5 characters long)

Reply:
fixed to be `otherOp`